### PR TITLE
More resolver stuff

### PR DESCRIPTION
**Pull request by tdiede**

### CODE REVIEW

**Comment by tdiede** [comment]
The first commit here `3c3073e` is the same as in the other PR, but I believe if the other one is merged first, this one will update conflict-free.

File: src/clj/odin/graphql/resolvers/payment.clj [hunk 1]
```diff
@@ -170,16 +211,46 @@
...
+  [{:keys [teller] :as ctx}
+   {:keys [account property source source_types types
+           from to statuses currencies datekey] :as params}]
   (tb/assoc-when
    params
-   :statuses (when-some [xs statuses]
-               (map #(keyword "payment.status" (name %)) xs))
+   :customers (when-some [a account]
+                [(tcustomer/by-account teller a)])
+   ;; TODO how do these get passed in the plural? can only one property be sent to `parse-gql-params`
```
Comment (jalehman) [inline src/clj/odin/graphql/resolvers/payment.clj]: They don't right now, but we can ultimately change the schema to allow that. Not needed yet.

File: src/clj/odin/graphql/resolvers/payment.clj [hunk 2]
```diff
@@ -170,16 +211,46 @@
...
+   :properties (when-some [p property]
+                     ;; TODO is this the community code?
```
Comment (jalehman) [inline src/clj/odin/graphql/resolvers/payment.clj]: Yeah, it's the community's datomic id, so this is correct.

File: src/clj/odin/graphql/resolvers/payment.clj [hunk 3]
```diff
@@ -170,16 +211,46 @@
...
+              [(tsource/by-id teller s)])
+   :source-types (when-some [xs source_types]
+                   (map #(keyword "payment-source.type" (name %)) xs))
    :types (when-some [xs types]
             (map #(keyword "payment.type" (name %)) xs))
-   :customers (when-some [a account]
-                [(tcustomer/by-account teller a)])))
+   :statuses (when-some [xs statuses]
+               (map #(keyword "payment.status" (name %)) xs))
+   (:currency (name (first currencies)))))
```
Comment (jalehman) [inline src/clj/odin/graphql/resolvers/payment.clj]: This looks fishy...where's the key?

Comment (tdiede) [reply src/clj/odin/graphql/resolvers/payment.clj]: `:currency` is singular in `tpayments/query` `query-params` -- should we keep it singular on the graphql side?

Comment (jalehman) [reply src/clj/odin/graphql/resolvers/payment.clj]: Then it should be `:currency (name (first currencies))`, yeah? No parens?

**Pull request merged**